manifests/dynflow/worker.pp

```diff
@@ -11,15 +11,18 @@
   Boolean $service_enable = $::foreman::jobs_service_enable
 ) {
 
+  if $::foreman::db_manage {
+    Class['foreman::database::postgresql'] ~> Service["dynflow-sidekiq@${service_name}"]
```

Comment:
I think this chaining is better with `foreman::database` because then it starts after `db:migrate` and `db:seed` as well. It also has the conditional on `db_manage`. So you can just use `subscribe => Class['foreman::database::postgresql']` on the `service`